pkg/controllers/management/auth/indexes.go

```diff
@@ -37,20 +37,24 @@ func rbRoleSubjectKeys(roleName string, subjects []v1.Subject) []string {
 }
 
 func indexByMembershipBindingOwner(obj interface{}) ([]string, error) {
-	if obj, ok := obj.(runtime.Object); ok {
-		meta, err := meta2.Accessor(obj)
-		if err != nil {
-			return nil, err
-		}
+	obj, ok := obj.(runtime.Object)
+	if !ok {
+		return []string{}, nil
+	}
 
-		for k, v := range meta.GetLabels() {
-			if v == membershipBindingOwner {
-				return []string{meta.GetNamespace() + "/" + k}, nil
-			}
+	meta, err := meta2.Accessor(obj)
+	if err != nil {
+		return nil, err
+	}
+
+	var keys []string
+	for k, v := range meta.GetLabels() {
+		if v == membershipBindingOwner {
+			keys = append(keys, meta.GetNamespace()+"/"+k)
```

Comment:
nit: we can save namespace returned by meta.GetNamespace() before the for loop and use it?